doc/nrf/drivers/entropy_cc310.rst

```diff
@@ -7,18 +7,19 @@ CC310 entropy driver
    :local:
    :depth: 2
 
-You can use the CC3xx entropy driver (entropy_cc310) to generate random data using the Arm CryptoCell CC310 hardware.
-This hardware is available on the nRF52840 and nRF9160 devices.
+You can use the CC3xx entropy driver (*entropy_cc310*) to generate random data using the Arm CryptoCell CC310 hardware available on the following Nordic SoCs:
 
-When used on nRF52840, the entropy_cc3xx driver gathers entropy by using the CC310 hardware through the :ref:`nrf_cc3xx_platform_readme`.
+* nRF52840
+* nRF5340
+* nRF9160
 
-When used on nRF9160, the driver behavior depends on whether it is used in a secure or non-secure application:
+When used in conjunction with the nRF52840, the entropy_cc3xx driver gathers entropy by using the CC310 hardware through the :ref:`nrf_cc3xx_platform_readme`.
```

Comment:
- I would rather say: When used with nRFxxx
- I believe we don't need the definite article in front of the product name -> nRF52840 etc.